cellxpredict/visualization.py

```diff
@@ -1,37 +1,39 @@
+from typing import List
+
 import numpy as np
 from skimage.filters import median as median_filter
 from skimage.morphology import disk
 
-from typing import List
 
 def float_to_uint8(input: np.ndarray[np.float32]) -> np.ndarray[np.uint8]:
-    '''
+    """
     Transforms an np.float32 image into a np.uint8 image
 
     Parameters
     ----------
     input: np.array[np.float32]
         - Input image of shape (width,height,channels)
-    
+
     Returns
     -------
     output: np.array[np.uint8]
         - Output image of shape (width,height,channels)
-    '''
+    """
     mn = input.min()
     mx = input.max()
     mx -= mn
-    output = ((input - mn)/mx) * 255
+    output = ((input - mn) / mx) * 255
     return output.astype(np.uint8)
 
+
 def norm_channel(
     input: np.ndarray,
```

Comment:
```suggestion
    x: np.ndarray,
```

`input` is a protected function so we should change this to another parameter, e.g. `x` - see #16